consumer/src/main/kotlin/streams/StreamsTopicService.kt

```diff
@@ -0,0 +1,38 @@
+package streams
+
+import org.neo4j.kernel.impl.core.EmbeddedProxySPI
+import org.neo4j.kernel.impl.core.GraphPropertiesProxy
+import org.neo4j.kernel.internal.GraphDatabaseAPI
+import java.lang.IllegalArgumentException
+
+
+private const val STREAMS_TOPIC_KEY: String = "streams.sink.topic."
+
+class StreamsTopicService(private val db: GraphDatabaseAPI, private val streamsSinkConfiguration: StreamsSinkConfiguration) {
+    private val properties: GraphPropertiesProxy = db.dependencyResolver.resolveDependency(EmbeddedProxySPI::class.java).newGraphPropertiesProxy()
+
+    fun set(topic: String, query: String) {
+        db.beginTx().use {
```

Comment:
we have to make sure to only call this on a writeable instance.

Reply:
there is only 1 commit attached to this PR perhaps you pushed to a wrong branch?